beacon-chain/p2p/types/BUILD.bazel

```diff
@@ -0,0 +1,25 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_test")
+load("@prysm//tools/go:def.bzl", "go_library")
+
+go_library(
+    name = "go_default_library",
+    srcs = ["types.go"],
+    importpath = "github.com/prysmaticlabs/prysm/beacon-chain/p2p/types",
+    visibility = ["//visibility:public"],
```

Comment:
Should the visibility be public here?

Reply:
nice catch ! that is why the CI was complaining lol